components/browser/state/src/main/java/mozilla/components/browser/state/state/SessionState.kt

```diff
@@ -24,6 +25,7 @@ interface SessionState {
     val engineState: EngineState
     val extensionState: Map<String, WebExtensionState>
     val contextId: String?
+    var lastAccess: Long
```

Comment:
I don't think we need this on the `SessionState`. I'm not sure what Custom tabs would get out of this if we have a `lastAccess` attribute there. 🤔 

It might better leaving this on only the `TabSessionState` for now and we can always add it to Custom Tabs later if we need it.